introduction/modules/modules.md

```diff
@@ -145,9 +145,12 @@ read by other users.
 
 ### Outbox
 
-The outbox module contains an implementation of the [transactional outbox
+The `outbox` module contains an implementation of the [transactional outbox
 pattern](https://microservices.io/patterns/data/transactional-outbox.html).
 
+### Outbox RabbitMQ
+The `outbox-rabbitmq` module adds support for publishing `outbox` messages to RabbitMQ (AMQP 0.9.1).
```

Comment:
Can you link to https://www.rabbitmq.com/ somewhere for an explanation on what it is?